### PR TITLE
834788-Prepare UG content for new features in SfTextInputLayout and SfMaskedEntry control

### PR DESCRIPTION
**Pull request by AarthiArjunan**
Description:
I have added the SfNumericEntry, SfPicker, SfTimePicker, and SfDatePicker feature content to the Supported Input Views pages of SfTextInputLayout.

### CODE REVIEW

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 1]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
 {% endhighlight %}
 {% endtabs %}
 
-![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
\ No newline at end of file
+![MaskedEntry](images/SupportedInputViews/MaskedEntry.png)
+
+## Picker
+
+To initialize the `Picker` control and launch it in each platform, refer to the `getting started with picker` documentation.
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:191]: Add Picker documentation (https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker)

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 2]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+{% endhighlight %}
+{% endtabs %}
+
+![Picker](images/SupportedInputViews/Picker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms Picker` as input view of the text input layout.
+
+## TimePicker
+
+To initialize the `TimePicker` control and launch it in each platform, refer to the `getting started with time picker` documentation.
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:229]: Add time picker documentation link: https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 3]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+inputLayout.Content = new TimePicker() {Time="12:00"}; 
+
+{% endhighlight %}
+{% endtabs %}
+
+![TimePicker](images/SupportedInputViews/TimePicker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms TimePicker` as input view of the text input layout.
+
+## DatePicker
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:257]: Add time date picker documentation link: https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/DatePicker

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 4]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+
+{% endhighlight %}
+{% highlight C# %}
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Card number"; 
+inputLayout.HelperText = "Required *";
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new SfNumericEntry() {Value="100", ShowClearButton="True", UpDownPlacementMode="Inline"};
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:311]: Recheck the C# code for numeric entry object creation.

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 5]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+inputLayout.Content = new DatePicker() {Date="06/20/2023"}; 
+
+{% endhighlight %}
+{% endtabs %}
+
+![DatePicker](images/SupportedInputViews/DatePicker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms DatePicker` as input view of the text input layout.
+
+## NumericEntry
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:287]: Place the NumericEntry UG content below the MaskedEntry UG content

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 6]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new DatePicker() {Date="06/20/2023"}; 
+
+{% endhighlight %}
+{% endtabs %}
+
+![DatePicker](images/SupportedInputViews/DatePicker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms DatePicker` as input view of the text input layout.
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:285]: N> Windows platform will not support `.NET MAUI DatePicker` as input view of the text input layout.

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 7]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new TimePicker() {Time="12:00"}; 
+
+{% endhighlight %}
+{% endtabs %}
+
+![TimePicker](images/SupportedInputViews/TimePicker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms TimePicker` as input view of the text input layout.
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:255]: N> Windows platform will not support `.NET MAUI TimePicker` as input view of the text input layout.

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 8]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new SfPicker(); 
+
+{% endhighlight %}
+{% endtabs %}
+
+![Picker](images/SupportedInputViews/Picker.jpg)
+
+N> UWP and WPF will not support `Xamarin.Forms Picker` as input view of the text input layout.
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:225]: N> Windows platform will not support `.NET MAUI Picker` as input view of the text input layout.

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 9]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+
+{% endhighlight %}
+{% highlight C# %}
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Fruit"; 
+inputLayout.HelperText = "Select a fruit";
+inputLayout.ContainerType = ContainerType.OutLined;
+inputLayout.ContainerBackground = Colors.Transparent;
+inputLayout.Content = new SfPicker();
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:218]: Picker object initialization is different from XAML code. ItemsSource and Selection Index properties are missing

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 10]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+
+To initialize the `DatePicker` control and launch it in each platform, refer to the `getting started with date picker` documentation.
+
+{% tabs %}
+{% highlight XAML %}
+
+<inputLayout:SfTextInputLayout Hint="Date of Birth" 
+                               HelperText="Select birth date"
+                               ContainerType="Outlined" >
+    <datepicker:DatePicker Date="06/20/2023"/>
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:267]: Remove the Date value in both C# and Xaml code snippets

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 11]
```diff
@@ -184,4 +184,133 @@ inputLayout.Content = new SfMaskedEntry() { MaskType = MaskedEntryMaskType.Simpl
...
+
+To initialize the `TimePicker` control and launch it in each platform, refer to the `getting started with time picker` documentation.
+
+{% tabs %}
+{% highlight XAML %}
+
+ <inputLayout:SfTextInputLayout Hint="Time" 
+                               HelperText="Select a start time"
+                               ContainerType="Outlined" >
+    <timepicker:TimePicker Time="12:00"/>
```
Comment (anandhan-rajagopal) [inline MAUI/TextInputLayout/Supported-input-views.md:237]: Remove the TimeValue in both C# and XAML code snippetes

**Review by anandhan-rajagopal — CHANGES_REQUESTED** [review]

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 1 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:191]: Addressed the changes

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 2 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:229]: Addressed the changes

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 3 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:257]: Addressed the changes.

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 5 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:287]: Addressed the changes.

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 6 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:285]: Addressed the changes.

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 7 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:255]: Addressed the changes.

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 8 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:225]: Addressed the changes.

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 9 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:218]: Addressed the changes.

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 10 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:267]: Addressed the changes

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 11 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:237]: Addressed the changes.

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 4 above)
Comment (AarthiArjunan) [reply MAUI/TextInputLayout/Supported-input-views.md:311]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 12]
```diff
@@ -0,0 +1,201 @@
...
+platform: maui
+control: SfMaskedEntry
+documentation: ug
+---
+
+# Basic Features in .NET MAUI Masked Entry (SfMaskedEntry)
+
+## Setting ClearButtonVisibility
+
+ClearButtonVisibility, of type ClearButtonVisibility, controls whether a clear button is displayed, which enables the user to clear the text. The default value of this property is `ClearButtonVisibility.Never`.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Basic-Features.md:14]: The ClearButtonVisibility of type ClearButtonVisibility controls whether a clear button is displayed, which enables the user to clear the text. The default value of this property is `ClearButtonVisibility.Never`.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 13]
```diff
@@ -0,0 +1,201 @@
...
+maskedEntry.ClearButtonVisibility = Visible;
+
+{% endhighlight %}
+{% endtabs %}
+
+![ClearButtonVisibility](MaskedEntry_Images/MaskedEntry_ClearButtonVisibility.png)
+
+## Setting CursorPosition
+
+Cursor position in the masked entry can either be obtained or updated using `CursorPosition` property in SfMaskedEntry.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Basic-Features.md:40]: The cursor position in the masked entry can either be obtained or updated using the `CursorPosition` property in SfMaskedEntry.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 14]
```diff
@@ -0,0 +1,201 @@
...
+
+## Setting FontSize
+The Masked Entry control provides the user to customize the font size of the text inside the entry part using `FontSize` property of type double.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Basic-Features.md:43]: The Masked Entry control allows the user to customize the font size of the text inside the entry part using the `FontSize` property of type double.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 15]
```diff
@@ -0,0 +1,201 @@
...
+maskedEntry.FontAttributes = FontAttributes.Bold;   
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontAttributes](MaskedEntry_Images/MaskedEntry_FontAttributes.png)
+
+## Setting FontFamily
+
+The Masked Entry control provides the user to customize the font family of the text inside the entry part using `FontFamily` property of type string.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Basic-Features.md:97]: The Masked Entry control allows the user to customize the text's font family inside the entry part using the `FontFamily` property of the type string.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 16]
```diff
@@ -0,0 +1,201 @@
...
+![FontFamily](MaskedEntry_Images/MaskedEntry_FontFamily.png)
+
+## Setting Keyboard
+
+The Masked Entry control provides the user to specifies the virtual keyboard that's displayed when entering text using `Keyboard` property of type Keyboard.
+
+
+## Setting Placeholder
+
+The Masked Entry control provides the user to defines the text that's displayed when the Mask and Value is empty using `Placeholder` property of type `string.` The default value of the Placeholder property is string.Empty.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Basic-Features.md:129]: The Masked Entry control allows the user to define the text displayed when the Mask and Value are empty using the `Placeholder` property of type `string.` The default value of the Placeholder property is string.Empty.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 17]
```diff
@@ -0,0 +1,201 @@
...
+maskedEntry.Placeholder = "Enter the value";   
+
+{% endhighlight %}
+{% endtabs %}
+
+![Placeholder](MaskedEntry_Images/MaskedEntry_Placeholder.png)
+
+## Setting PlaceholderColor
+
+The `SfMaskedEntry` control provides the user to customize the color of the placeholder text using `PlaceholderColor` property of type `Color.` The default value of the PlaceholderColor property is Colors.Gray.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Basic-Features.md:151]: The `SfMaskedEntry` control allows the user to customize the color of the placeholder text using the `PlaceholderColor` property of type `Color.` The default value of the PlaceholderColor property is Colors.Gray.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 18]
```diff
@@ -0,0 +1,201 @@
...
+maskedEntry.PlaceholderColor =  Colors.Green;   
+
+{% endhighlight %}
+{% endtabs %}
+
+![PlaceholderColor](MaskedEntry_Images/MaskedEntry_PlaceholderColor.png)
+
+## Setting TextColor
+
+The `SfMaskedEntry` control provides the user to customize the color of the entered text using `TextColor` property of type `Color.` The default value of the TextColor property is Colors.Black.
```
Comment (Chrispineagunja) [inline MAUI/Masked-Entry/Basic-Features.md:175]: The `SfMaskedEntry` control provides the user to customize the color of the entered text using the `TextColor` property of type `Color.` The default value of the TextColor property is Colors.Black.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 19]
```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Basic Features in .NET MAUI Masked Entry control | Syncfusion
+description:  Learn here all about Basic Features support in Syncfusion .NET MAUI Masked Entry (SfMaskedEntry) control and more.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:4]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 12 above)
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:14]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 13 above)
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:40]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 14 above)
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:43]: Addressed the changes

File: MAUI/Masked-Entry/Basic-Features.md [hunk 20]
```diff
@@ -0,0 +1,201 @@
...
+maskedEntry.FontSize = 18;   
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontSize](MaskedEntry_Images/MaskedEntry_FontSize.png)
+
+## Setting FontAttributes
+
+The Masked Entry control provides the user to customize the font attribute of the text inside the entry part using `FontAttributes` property of type FontAttributes.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:70]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 15 above)
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:97]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md [hunk 21]
```diff
@@ -0,0 +1,201 @@
...
+maskedEntry.FontFamily = "Lobster-Regular";  
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontFamily](MaskedEntry_Images/MaskedEntry_FontFamily.png)
+
+## Setting Keyboard
+
+The Masked Entry control provides the user to specifies the virtual keyboard that's displayed when entering text using `Keyboard` property of type Keyboard.
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:124]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 16 above)
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:129]: Addressed the changes.

File: MAUI/Masked-Entry/Basic-Features.md (hunk 17 above)
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:151]: Addressed the changes

File: MAUI/Masked-Entry/Basic-Features.md (hunk 18 above)
Comment (AarthiArjunan) [reply MAUI/Masked-Entry/Basic-Features.md:175]: Addressed the changes.

**Review by Choza-rajan — APPROVED** [review]

**Pull request closed without merging**